src/foraging_gui/Foraging.py

```diff
@@ -368,6 +370,16 @@ def _ConnectBonsai(self):
                 self.WarningLabelInitializeBonsai.setStyleSheet("color: red;")
                 self.InitializeBonsaiSuccessfully=0
 
+    def _ReconnectBonsai(self):
+        '''Reconnect bonsai'''
+        if self.InitializeBonsaiSuccessfully==1:
+            self.client.close()
```

Comment:
Why would we want to close the clients and reconnect?

Reply:
In some cases, the connection between GUI and Bonsai is interrupted by mistake and we can reconnect them. I also use it for testing when I want to change the bonsai code while the connection is already established so I don't need to reopen the GUI.